meta-luneos/recipes-webos-ose/webos-connman-adapter/webos-connman-adapter.bb

```diff
@@ -31,45 +32,17 @@ EXTRA_OECMAKE += "-DENABLE_SCAN_ON_SOFTAP=true"
-# webos-connman-adapter/1.1.0-43/git/src/connectionmanager_service.c:1410:74: error: passing argument 4 of 'change_network_ipv6' makes pointer from integer without a cast [-Wint-conversion]
-# webos-connman-adapter/1.1.0-43/git/src/connectionmanager_service.c:1563:93: error: passing argument 3 of 'change_network_dns' from incompatible pointer type [-Wincompatible-pointer-types]
-# webos-connman-adapter/1.1.0-43/git/src/connectionmanager_service.c:2469:47: error: passing argument 1 of 'connman_counter_new' from incompatible pointer type [-Wincompatible-pointer-types]
-# webos-connman-adapter/1.1.0-43/git/src/wifi_scan.c:113:44: error: implicit declaration of function 'is_connected_peer' [-Wimplicit-function-declaration]
-# webos-connman-adapter/1.1.0-43/git/src/wifi_scan.c:160:18: error: implicit declaration of function 'is_wifi_tethering' [-Wimplicit-function-declaration]
-# webos-connman-adapter/1.1.0-43/git/src/wifi_scan.c:434:40: error: passing argument 1 of 'g_strv_length' from incompatible pointer type [-Wincompatible-pointer-types]
-# webos-connman-adapter/1.1.0-43/git/src/wifi_scan.c:444:64: error: passing argument 1 of 'strlen' from incompatible pointer type [-Wincompatible-pointer-types]
-# webos-connman-adapter/1.1.0-43/git/src/wifi_scan.c:444:48: error: passing argument 2 of 'strncat' from incompatible pointer type [-Wincompatible-pointer-types]
-# webos-connman-adapter/1.1.0-43/git/src/wifi_service.c:1841:33: error: implicit declaration of function 'setPropertyUpdateCallback' [-Wimplicit-function-declaration]
-# webos-connman-adapter/1.1.0-43/git/src/wifi_service.c:2723:48: error: passing argument 1 of 'wifi_scan_now_with_option' from incompatible pointer type [-Wincompatible-pointer-types]
-# webos-connman-adapter/1.1.0-43/git/src/wifi_service.c:2723:54: error: passing argument 2 of 'wifi_scan_now_with_option' from incompatible pointer type [-Wincompatible-pointer-types]
-# webos-connman-adapter/1.1.0-43/git/src/wifi_service.c:2896:72: error: passing argument 2 of 'check_service_security' from incompatible pointer type [-Wincompatible-pointer-types]
-# webos-connman-adapter/1.1.0-43/git/src/wifi_service.c:4153:56: error: passing argument 2 of 'LSSubscriptionSetCancelFunction' from incompatible pointer type [-Wincompatible-pointer-types]
-# webos-connman-adapter/1.1.0-43/git/src/wifi_service.c:4162:82: error: passing argument 5 of 'g_bus_watch_name' from incompatible pointer type [-Wincompatible-pointer-types]
-# webos-connman-adapter/1.1.0-43/git/src/wifi_tethering_service.c:752:42: error: passing argument 2 of 'jboolean_get' from incompatible pointer type [-Wincompatible-pointer-types]
-CFLAGS += "-Wno-error=incompatible-pointer-types -Wno-error=int-conversion -Wno-error=implicit-function-declaration"
```

Comment:
implicit-function-declaration is still needed as it's broken in webos-ports by your
meta-luneos/recipes-webos-ose/webos-connman-adapter/webos-connman-adapter/0001-Add-back-com.palm.wan-for-cellular-support.patch

see:
webos-connman-adapter/1.1.0-50/git/src/wan_service.c:587:14: error: implicit declaration of function 'cellular_technology_status_check_with_subscription'; did you mean 'p2p_technology_status_check_with_subscription'? [-Wimplicit-function-declaration]

Reply:
Because this error is enabled by default since gcc-14, you're probably building with scarthgap with gcc-13. See bd8258b80b61caf39239f70f204b73817e023159

Reply:
In scarthgap you can grep the log.do_compile to see the warning (it's just not fatal for the build as it is with gcc-14)